prez/content/juice-shop-hints.md

```diff
@@ -3,74 +3,77 @@
-6. Get it to return as TRUE (sql) or find the admin's name
-7. Now we know the SQL query related to logging in. We can send **'** OR TRUE -- as email input and any string as a password
-8. "SELECT * FROM Users WHERE email = '' OR TRUE -- AND password = '111' ..."
-9. Observe the admin user name in the response - admin@juice-sh.op
-10. Also take note on the SQL that is used (from the error response)
+1. Let's check if the website is adding our input to the SQL
+1. Test with email: **'** e.g. pwd: 111
+1. Check Network tab in your browser and "Response" for the 500 error. There is a json with the SQL used
+1. "SELECT * FROM Users WHERE email = '**'**' AND password = '111' ..."
+1. Get it to return as TRUE (sql) or find the admin's name
+1. Now we know the SQL query related to logging in. We can send **'** OR TRUE -- as email input and any string as a password
+1. "SELECT * FROM Users WHERE email = '' OR TRUE -- AND password = '111' ..."
+1. Observe the admin user name in the response - admin@juice-sh.op
+1. Also take note on the SQL that is used (from the error response)
 
 </code>
 
 ---
+
 ## A01:Admin Section
 
->Challenge: Access the administration section of the juice shop<!-- .element: style="font-size:0.8em"-->
+>Challenge: Access the administration section of the juice shop <!-- .element: style="font-size:0.8em"-->
 
 <code style="font-size: 0.5em;">
 
-0. Do not log-in to the juice-shop
+1. Do not log-in to the juice-shop
 1. Find the administrator page
-2. User the debugger TAB look in the source => main.js
-3. Search for admin - and we will find 'administration'
-4. Remember to login as `admin` before you try the URL
+1. User the debugger TAB look in the source => main.js
+1. Search for admin - and we will find 'administration'
+1. Remember to login as `admin` before you try the URL
 
 </code>
 
 ---
+
 ## A02:Weird Crypto
 
 >Inform the shop about an algorithm or library it should definitely not use the way it does.<br>http://&lt;yourhost&gt;:3000/#/contact
 
 <code style="font-size: 0.5em;">
 
 1. Create a new user
-2. Change password for new user with an easy one e.g 'admin123' and observe response from /change-password? (In the Firefox tool)
-3. e.g. password:"0192023a7bbd73250516f069df18b500"
-4. What type of hash is this?
-5. Use this information (hash type) to give feedback to juice-shop
-6. http://&lt;yourhost&gt;/#/contact
-7. You can also solve this by googling - **weak crypto algorithms** 
-8. Hint on 4 - **hash analyzer tunnelsup**
+1. Change password for new user with an easy one e.g 'admin123' and observe response from /change-password? (In the Firefox tool)
+1. e.g. password:"0192023a7bbd73250516f069df18b500"
+1. What type of hash is this?
+1. Use this information (hash type) to give feedback to juice-shop
+1. http://&lt;yourhost&gt;/#/contact
+1. You can also solve this by googling - **weak crypto algorithms**
+1. Hint on 4 - **hash analyzer tunnelsup**
 
```

Comment:
Changed to 1 to avoid having to manually enumerate each number. They will automatically get the correct number, so you avoid any manual enumeration in the future.